### PR TITLE
Vertex attributes cleanup

### PR DESCRIPTION
**Pull request by kvark**
This is the preparation work for #457

---
This change is [Reviewable](https://reviewable.io/reviews/servo/webrender/610)

### CODE REVIEW

File: webrender/res/debug_color.vs.glsl [hunk 1]
```diff
@@ -2,6 +2,9 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+in vec4 aColorRectTL;
```
Comment (glennw) [inline webrender/res/debug_color.vs.glsl]: Let's make this just be aColor

File: webrender/res/debug_font.vs.glsl [hunk 2]
```diff
@@ -2,6 +2,12 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+in vec4 aColorRectTL;
+in vec4 aColorTexCoordRectTop;
```
Comment (glennw) [inline webrender/res/debug_font.vs.glsl]: Let's just have aColor and aUv0 or something similar.

File: webrender/src/internal_types.rs [hunk 3]
```diff
@@ -225,21 +225,8 @@ pub const DEFAULT_TEXTURE: TextureSampler = TextureSampler::Color0;
 
 pub enum VertexAttribute {
     Position,
-    PositionRect,
     ColorRectTL,
```
Comment (glennw) [inline webrender/src/internal_types.rs]: Let's rename these to Color and Uv0 or something like that.

File: webrender/src/renderer.rs [hunk 4]
```diff
@@ -308,7 +308,7 @@ fn create_clear_shader(name: &'static str,
                        max_ubo_vectors: usize) -> ProgramId {
     let prefix = format!("#define WR_MAX_UBO_VECTORS {}", max_ubo_vectors);
 
-    let includes = &["shared_other"];
```
Comment (glennw) [inline webrender/src/renderer.rs]: Should we delete shared_other.glsl - is it ever used now?

Comment (kvark) [inline webrender/src/renderer.rs]: That's what I did first, but then - `Device::create_program` requires a shared header, and so I decided to let it live for now :)

File: webrender/res/debug_color.vs.glsl (hunk 1 above)
Comment (kvark) [inline webrender/res/debug_color.vs.glsl]: Yep, I have it on my list. Clearly, TL/RectTop suffices are misleading with this change

**Comment by kvark** [comment]
Should be ready now.

**Pull request merged**